api-authorizations/pom-template.xml

```diff
@@ -27,9 +30,71 @@
       </plugin>
       <plugin>
         <artifactId>maven-resources-plugin</artifactId>
+        <executions combine.children="append">
+          <execution>
+            <id>copy-elm-toserve</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>target/classes/toserve/${project.artifactId}</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>src/main/elm/toserve</directory>
+                  <includes>
+                    <include>*.css</include>
+                    <include>*.js</include>
+                  </includes>
+                </resource>
+                <resource>
+                  <directory>src/main/elm/generated</directory>
+                  <includes>
+                    <include>*.css</include>
+                    <include>*.js</include>
+                  </includes>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-elm-template</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>target/classes/template</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>src/main/elm</directory>
+                  <includes><include>reportingManagement.html</include></includes>
```

Comment:
I think this file is missing and it should be renamed according to correct plugin name

Reply:
In fact we do not need that section about the html template we will include an plugin on api account page